### DISCUSSION TITLE
Need help for setting up Keycloakify on existing react project

### DISCUSSION THREAD

**Discussion by pranitkhadilkar7** [keycloakify/keycloakify#435] [Q&A]
I have existing react project in which I have added keyclokify
I have followed instructions given in https://docs.keycloakify.dev/keycloakify-in-my-app. 
So, inside src directory I have created new directory named keycloak-theme similar to what we have in keycloak-starter project.
When I was trying to build the theme I was able to get the respective jar file in the build_keycloak/target. But when I use this theme inside keycloak I'm getting error rendering login.ftl<img width="1440" alt="Screenshot 2023-10-18 at 4 08 55 PM" src="https://github.com/keycloakify/keycloakify/assets/98008346/6ec67f2b-ef32-495f-8090-027a824f379b">

**Comment by garronej** [comment]
Hello @pranitkhadilkar7,  
When you use the starter project, does it work?  
What version of Keycloak are you using?  
What does the Keycloak logs says?
